machinelearning/tensorflow.md

```diff
@@ -30,14 +30,14 @@ docker run -p 8501:8501 --name tfserving_resnet --mount type=bind,source=/tmp/re
 pip install tensorflow
 ```
 
-TensorFlow wheel supports the default eigen backend for Graviton CPUs, but typically onednn+acl provides better performance and this can be enabled by setting the below TF environment variable
+Starting from version 2.14.0, TensorFlow wheel enabled onednn+acl backend by default on Graviton CPUs which typically provides better performance than eigen backend. For previous versions of TensorFlow, onednn+acl can be enabled by setting the below TF environment variable
```

Comment:
I would reword this sentence for better reading.
"On Graviton CPUs, onednn+acl is the default backend in TensorFlow v2.14.0 wheel. This backend typically provides better performance than eigen backend. For prev..."